### PR TITLE
Add RUNTIME_DESTINATION variable

### PR DESCRIPTION
**Pull request by dilshodm**
Currently libs are installed under `<...>/lib/<packagename-version>/` directory. 

As a user I want to be able to configure relative subdirectory, similar to `INCLUDE_DESTINATION`.

For example, I want to install mylib.so as `/usr/lib/mylib.so` instead of `/usr/lib/mylib-1.0/mylib.so`. In this case I have to set `RUNTIME_DESTINATION "/"` as relative path.

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -68,6 +68,8 @@ jobs:
           cpack -G DEB .
           dpkg -I *deb | grep "Maintainer: Foo Bar <foo@bar.local>"
           sudo -E dpkg -i *.deb
+          echo "Testing existense of libruntime_destination_dependency.a in /usr/lib"
+          test -e /usr/lib/libruntime_destination_dependency.a
```
Comment (TheLartians) [inline .github/workflows/test.yml:72]: The check is not related to the packaging process. Would it make sense to move it into a separate step instead?

```suggestion
      - name: check if we can explicitly set the runtime destination
      - run: test -e /usr/lib/libruntime_destination_dependency.a
```

**Review by TheLartians — CHANGES_REQUESTED** [review]
Thanks for adding this feature and a test case! Looks great, just a small suggestion to keep the CI script steps more organised.

File: .github/workflows/test.yml (hunk 1 above)
Comment (dilshodm) [reply .github/workflows/test.yml]: Thank you for the review!

**Review by TheLartians — APPROVED** [review]
Thanks for the update!

**Comment by TheLartians** [comment]
Feature released in `v1.13.0`! 🎉

**Pull request merged**